### ISSUE TITLE
Start and End times seem backwards

### ISSUE THREAD

**Issue by jlgaustin** [joedaniels29/10Clock#10]
Sleep increments the end time and wake increments the start time.   Intuitively this is backwards.

**Comment by joedaniels29** [comment]
Yeah, I think in this repo its somehow ended up backwards. It needs a modest rework, but its not enough to change just the names of the variables, because originally, the logic was meant to go the other way, but one or two bugs ended up piling up, and one or two wrong solutions were proposed, and now its totally reversed. haha. Feel free to submit a PR if you want to comb through the code a bit to fix. 

thanks for reporting
